watertap/unit_models/electrodialysis_1D.py

```diff
@@ -64,6 +64,25 @@ class ElectricalOperationMode(Enum):
     Constant_Voltage = 1
 
 
+class PressureDropMethod(Enum):
+    none = 0
+    experimental = 1
+    Darcy_Weisbach = 2
+
+
+class FrictionFactorMethod(Enum):
+    defined = 0
+    Ponzio = 1
+    Gurreri = 2
+    Kuroda = 3
```

Comment:
Very nice!

Reply:
Should the new options also be in 0D?

Reply:
Yes if we want (I say this just because it looks like 1D will be selected for most use cases).  I am thinking a separate PR for 0D to mirror changes in 1D,  after the 1D unit works for the new flowsheet under development to our satisfaction.

Reply:
We definitely should for consistency. I agree that this can be done in a subsequent PR.